pages/cloud/storage/object_storage/s3_getting_started_with_cold_archive/guide.de-de.md

```diff
@@ -1,14 +1,12 @@
 ---
-title: Cold Archive - Getting started with Cold Archive (Alpha) (EN)
+title: Cold Archive - Getting started with Cold Archive (Beta)
 slug: s3/getting-started-with-cold-archive
-routes:
```

Comment:
Please re-integrate the canonical link on the subs where it's deleted, replacing the link with https://docs.ovh.com/gb/en/storage/object-storage/s3/getting-started-with-cold-archive/